### PR TITLE
[SMALLFIX]Javadoc for ContainerAllocator#getPotentialWorkerHosts

### PR DESCRIPTION
**Pull request by ParanoiaW**

### CODE REVIEW

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 1]
```diff
@@ -87,6 +87,11 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * Gets the hosts of workers that could be allocated.
+   *
+   * @return the unused worker hosts
```
Comment (aaudiber) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: "@return a list of all hosts in the cluster which haven't yet reached the containers per host limit."

Comment (ParanoiaW) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: thanks for your comments and i have modified the @return sentence.

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 2]
```diff
@@ -87,6 +87,11 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * Gets the hosts of workers that could be allocated.
```
Comment (aaudiber) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: The `@return` holds all the information, so we don't need this part

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 3]
```diff
@@ -87,6 +87,11 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
...
+   *
+   * @return a list of all hosts in the cluster which haven't yet reached the containers per host limit.
```
Comment (aaudiber) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: I think this line might be over 100 characters

Also, there can't be a period at the end of `@return`

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 4]
```diff
@@ -87,6 +87,10 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * @return a list of all hosts in the cluster which haven't yet reached
+   * the containers per host limit.
```
Comment (aaudiber) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: The text should be indented to line up with "a list of ...", see WriteType.java for examples of javadoc line wrapping style

There should be no period at the end

File: integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java [hunk 5]
```diff
@@ -87,6 +87,10 @@ public ContainerAllocator(String containerName, int targetNumContainers, int max
     mAllocatedContainers = new ArrayList<>();
   }
 
+  /**
+   * @return a list of all hosts in the cluster which haven't yet reached
+   * the containers per host limit
```
Comment (aaudiber) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: Please indent as

```
/**
 * @return a list of all hosts in the cluster which haven't yet reached
 *         the containers per host limit
 */
```

Comment (ParanoiaW) [inline integration/yarn/src/main/java/alluxio/yarn/ContainerAllocator.java]: thanks for your comment and the change has been done

**Pull request merged**